app/src/main/java/ru/netology/fmhandroid/adapter/NoteListAdapter.kt

```diff
@@ -0,0 +1,95 @@
+package ru.netology.fmhandroid.adapter
+
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import ru.netology.fmhandroid.R
+import ru.netology.fmhandroid.databinding.NoteCardItemForListBinding
+import ru.netology.fmhandroid.domain.BusinessRules
+import ru.netology.fmhandroid.dto.Note
+import ru.netology.fmhandroid.enum.ExecutionPriority
+import ru.netology.fmhandroid.utils.Utils
+import java.time.LocalDateTime
+
+interface OnNoteItemClickListener {
+    fun onCard(note: Note) {}
```

Comment:
Лучше не использовать в интерфейсе реализацию по умолчанию - тот ещё хак
```suggestion
    fun onCard(note: Note)
```

Reply:
Исправлено